events/Organizations/InviteAccountToOrganization.json

```diff
@@ -9,6 +9,31 @@
     "mitreAttackTechniques": [
         "T1535 - Unused/Unsupported Cloud Regions"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1098 - Account Manipulation",
+            "reason": "Adding accounts to the organization can be used to manipulate account permissions and roles for persistence or escalation of privileges."
+        },
+        {
+            "technique": "T1071 - Application Layer Protocol",
+            "reason": "Using the API call to communicate and manage accounts in the organization can evade detection mechanisms focused on other layers."
```

Comment:
I think this is a bit confusing. Because in the end all API calls can be considered Application Layer Protocol.

Reply:
Removed.